### ISSUE TITLE
Accessibility - Web - Pages Missing Language

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#91]
In the web pages, there is not even a default language set for the pages which should be English (lang="en"). By not having a language set or the wrong one will cause issues for screen reader users. Say I'm a Spanish speaking user and I come to your page that doesn't have language set. JAWS will read your English words with a Spanish accent really fast.

When a person changes languages to either Spanish or French, the pages should be set to that language for all pages in the application until the person switches to something else. Which most likely won't happen ever again for others or often, but because I'm testing all the languages mine should keep changing.

So no Spanish or French accents for the proper pages. VoiceOver is reading "2 / 8" as "two slash eight" on the Spanish page instead of "dos barras ocho" or "deux slash huit" for the French page. Please note my translations came from Google translate. So please get any other messages in the application appropriately translated.

**Comment by sanganalakshay** [comment]
fixed

**Comment by jfc3-dol** [comment]
Nice start by adding a language set to the pages, but no matter which language you have picked the language of the page is English ( lang="en" ).

**Comment by sanganalakshay** [comment]
Can you check to see if you have the latest version of code downloaded? I pushed another fix today morning.  
When I changed to French and inspected the page I found it to correctly display lang='fr'. 
Is this what you are referring to ?


![capture](https://user-images.githubusercontent.com/15368456/43420414-9797d55c-9411-11e8-89d6-4791670a5261.PNG)

**Comment by jfc3-dol** [comment]
DEVTOOLS inspect shows the correct language but when I "View Source" it's in English.

Will check with a screen reader to see what language the page is read in but have an issue that I can only look at WEB pages in Chrome and my screen reader works best in IE or Firefox.

See the screenshot below.

![complychain - web - spanish - but view source shows english](https://user-images.githubusercontent.com/10423487/43421887-04943c3c-9416-11e8-9a3a-45e5a2300465.png)

**Comment by sanganalakshay** [comment]
Yeah the source has lang set to "en" for default purposes but if the language is changed or if the language is not English, then the lang is dynamically changed using JavaScript to match the correct language

**Comment by jfc3-dol** [comment]
Thanks, @sanganalakshay will give it a test with my screen reader if I can since JAWS doesn't work so well with Chrome. For some reason with the latest update, I can't use IE or Firefox to view any of the pages which will make testing difficult because my tools are in those browsers.

**Comment by jfc3-dol** [comment]
When I used VoiceOver on my iPad, it read in the proper language that I had chosen. Once the newest build gets deployed and I can use JAWS on IE or Firefox I will close this issue if it to works which I'm expecting it to do so.

**Comment by jfc3-dol** [comment]
JAWS is reading the English, Spanish, and French pages as expected on IE 11.

I'm going to close this issue.
